tests/e2e/state_machine_coverage.py

```
@@ -0,0 +1,487 @@
+    auth_state: AuthState
+    policy_state: PolicyState
+    operation_class: OperationClass
+    provider_outcome: ProviderOutcome
+    lifecycle_state: LifecycleState
+    delivery_target: JourneyDeliveryTarget
+    sequences: tuple[SequenceClass, ...]
+    invariants: tuple[StateMachineInvariant, ...]
+    evidence: Evidence
+
+
+def _cargo(source: str, test: str, target: str, manifest: str | None = None):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_cargo`

(ANN202)